package.json

```diff
@@ -486,6 +486,11 @@
           "type": "boolean",
           "default": false,
           "description": "Enable R session watcher (experimental). Restart required to take effect."
+        },
+        "r.rtermSendDelay": {
+          "type" : "integer",
+          "default" : 8,
+          "description": "Delay before sending text to rterm"
```

Comment:
Should we add the info that this only works with `r.bracketedPaste` disabled and it is in millisecond?

Reply:
Changed the description to:

```
Delay in milliseconds before sending each line to rterm (only applies if r.bracketedPaste is false)
```

Which hopefully should make it super clear what the parameter affects (and when)